### PR TITLE
maintainers: add @Yutyo

### LINKED ISSUE #5345: add collaborator: @Yutyo
Hi, @Yutyo! You seem to be enjoying contributing to the tldr-pages project.
You now have had five distinct pull requests merged (#5341, #5336, #5334, #5333, #5332, and a lot more)!
That qualifies you to become a collaborator in this repository, as explained in our [community roles documentation](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md).

As a collaborator, you will have commit access to the repository.
That means you can merge pull requests, label and close issues, and perform various other maintenance tasks that are needed here and there.
Of course, all of this is voluntary — you're welcome to contribute to the project in whatever ways suit your liking.

If you do decide to start performing maintenance tasks, though, we only ask you to get familiar with the [maintainer's guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md).

So, what do you say? Can we add you as a collaborator?

Either way, thanks for all your work so far!

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Close #5345

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ If you are an owner of the organization, you should be able to see an automated
   [24 August 2020](https://github.com/tldr-pages/tldr/issues/4291) — [05 October 2020](https://github.com/tldr-pages/tldr/issues/4504)
 - bl-ue ([@bl-ue](https://github.com/bl-ue)):
   [01 January 2021](https://github.com/tldr-pages/tldr/issues/5056) — [02 February 2021](https://github.com/tldr-pages/tldr/issues/5219)
+- Tan Siret Akıncı ([@yutyo](https://github.com/yutyo)):
```
Comment (bl-ue) [inline MAINTAINERS.md:56]: Your actual GitHub handle uses a captial 'Y'&mdash;wouldn't you want to use that?
```suggestion
- Tan Siret Akıncı ([@Yutyo](https://github.com/Yutyo)):
```

**Review by navarroaxel — APPROVED** [review]
Looks good after the comments from @bl-ue get addressed.

**Review by bl-ue — APPROVED** [review]
Welcome aboard @Yutyo! 🚀

**Comment by bl-ue** [comment]
Darn it!  I should have preview it first.  :laughing: It's in the wrong section.  Let me fix it...

**Pull request merged**